lib/static/components/controls/browser-list/index.jsx

```diff
@@ -5,224 +5,172 @@ import {flatten, isEmpty, get, chain, compact} from 'lodash';
-            }
-
-            return acc;
-        }, {});
-
-        const elementsNew = [];
-
-        for (const browserId in selectedBrowsers) {
-            const versions = selectedBrowsers[browserId];
-            if (!versions.length || (availableVersionsCount[browserId] === versions.length)) {
-                elementsNew.push(browserId);
-            } else {
-                elementsNew.push(...versions.map(version => buildComplexId(browserId, version)));
-            }
+    const renderOption = (option) => {
+        const isSelected = selected.includes(option.value);
+        const selectOnly = (e) => {
+            e.preventDefault();
+            e.stopPropagation();
+            setSelected([option.value]);
         }
-
-        return elementsNew;
-    }
-
-    function updateLabels() {
-        const newElements = buildElements();
-
-        if (get(elements, 'length', 0) !== 1 && get(newElements, 'length', 0) !== 1) {
-            setElements(newElements);
-            return;
+        const selectExcept = (e) => {
+            e.preventDefault();
+            e.stopPropagation();
+            setSelected(allOptions.filter(o => o != option.value));
         }
-
-        const updatingValues = [
-            get(newElements, 'length', undefined) === 1 && newElements[0],
-            get(elements, 'length', undefined) === 1 && elements[0]
-        ];
-
-        const nodesFlat = nodes.reduce((arr, node) => {
-            const children = node.children || [];
-
-            arr.push(node);
-            children.forEach(child => arr.push(child));
-
-            return arr;
-        }, []);
-
-        const updatingNodes = nodesFlat.filter(node => {
-            const {browserId, version} = node.data;
-
-            return updatingValues.includes(buildComplexId(browserId, version));
-        });
-
-        updatingNodes.forEach(node => {
-            const {browserId, version} = node.data;
-            node.label = mkLabel({browserId, version, elements: newElements});
-        });
-
-        setElements(newElements);
-    }
-
-    function formatSelectedData(selectedItems) {
-        return chain(selectedItems)
-            .reduce((acc, value) => {
-                const {browserId, version} = treeDataMap[value] || {};
-
-                if (browserId) {
-                    acc[browserId] = acc[browserId] || [];
-                    acc[browserId].push(version);
-                }
-
-                return acc;
-            }, {})
-            .map((versions, id) => ({id, versions: compact(versions)}))
-            .value();
-    }
-
-    if (isEmpty(available)) {
-        return (<div></div>);
+        return (
+        <div className='browserlist__row'>
+            <div className='browserlist__row_content'>
+            {option.content}
+            </div>
+            <Button size='s' onClick={isSelected ? selectExcept : selectOnly} className='action-button'>{isSelected ? 'Except' : 'Only'}</Button>
```

Comment:
"Only" should be displayed on both: selected and unselected options.
"Except" only should be displayed if it is the **only** selected option.

In your case, if only one option is selected, it would still has "Only" button, though it should have "Except"